lib/galaxy/web/base/interactive_environments.py

```diff
@@ -293,4 +296,4 @@ def get_proxied_ports(self, container_id):
                     binding['HostIp'],
                     binding['HostPort']
                 ))
-        return mappings
+        return mappings, gateway_ip
```

Comment:
A future version of this should probably split out the mapping adjustments into a separate method, and then just return `inspect_data` here.